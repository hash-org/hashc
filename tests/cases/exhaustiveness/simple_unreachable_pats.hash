// run=pass, warnings=compare

main := () => {
    a := 5;

    match a {
        3 => {};
        2 => {};
        3 => {};
        k => {};
        _ => {};
    }
};
