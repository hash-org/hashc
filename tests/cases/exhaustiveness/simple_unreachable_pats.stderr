warn: pattern is unreachable
  --> $DIR/simple_unreachable_pats.hash:9:9
 8 |           2 => {},
 9 |           3 => {},
   |           ^ 
10 |           k => {},

warn: pattern is unreachable
  --> $DIR/simple_unreachable_pats.hash:11:9
10 |           k => {},
11 |           _ => {}
   |           ^ 
12 |       }
